test/file.test.js

```diff
@@ -552,3 +553,719 @@ describe('file', function() {
         }).timeout(20000);
     });
 });
+
+describe('multipart upload concurrently', function () {
```

Comment:
I took all the tests here and changed the method called to the concurrent one: https://github.com/adobe/node-httptransfer/blob/586e992893e21a85f20f7552b63a7fff90bb7c4e/test/aemmultipart.test.js

I did the same thing for `uploadFileConcurrently` in a different commit: https://github.com/adobe/node-httptransfer/blob/0feae61f61d1ffe198d58e3c3ba69a0d011344c7/test/file.test.js#L408-L555

the reason for this is to ensure these functions can be used 1:1  for the previous functions in the pipeline: https://github.com/adobe/asset-compute-pipeline/blob/master/lib/storage/http.js#L53-L64
it also helped find edge case bugs